README.md

```diff
@@ -18,8 +18,8 @@ Please contact hello@codeclimate.com if you need any assistance setting this up.
 
 ## Usage
 
-```
-CODECLIMATE_REPO_TOKEN=my_token bundle exec rspec && bundle exec codeclimate-test-reporter
+```bash
```

Comment:
Sure sure, was trying to go with the best supported syntax highlighting. I'll switch this to `console`.